test/virtual/include/openenclave/edger8r/common.h

```diff
@@ -66,20 +66,33 @@ OE_INLINE oe_result_t oe_add_size(size_t* total, size_t size)
     return result;
 }
 
-#define OE_ADD_SIZE(total, size)                                   \
+#define OE_COMPUTE_SIZE(a, b, c)                 \
+    do                                           \
+    {                                            \
+        size_t _a = (size_t)(a);                 \
+        size_t _b = (size_t)(b);                 \
```

Comment:
We enforce the `_a` and `_b` to be `size_t`. Therefore, we only need one check (out of four) from https://github.com/openenclave/openenclave/blob/92d4ad31004c24c104968906b7505aeeb6f20459/include/openenclave/internal/safemath.h#L94.